examples/go/hello-world/README.md

```diff
@@ -0,0 +1,83 @@
+# Hello, world!
```

Comment:
@cwarnermm this one is really for you to review

Reply:
Thanks, @levb. From what I see here, you've made changes to the Go hello world code, and I can now test this sample app. Please let me know if I've misunderstood.